integration-tests/deployment/keystone/capability_management.go

```diff
@@ -0,0 +1,64 @@
+package keystone
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/logger"
+	"github.com/smartcontractkit/chainlink/integration-tests/deployment"
+	kcr "github.com/smartcontractkit/chainlink/v2/core/gethwrappers/keystone/generated/capabilities_registry"
+)
+
+// AddCapabilities adds the capabilities to the registry
+// it tries to add all capabilities in one go, if that fails, it falls back to adding them one by one
```

Comment:
Any reason why we wouldnt perform a  `reg.GetCapabilities(...)`, then create a set of capabilities that is the diff of what we're trying to add, and what already exists? It would mean that we'll only perform at most 1 tx every time this is invoked